tests/publiccloud/prepare_tools.pm

```diff
@@ -107,10 +107,6 @@ sub run {
     install_in_venv('az', requirements => 1);
     record_info('Azure', script_output('az -v'));
 
-    # Install OpenStack cli
-    install_in_venv('openstack', requirements => 1);
```

Comment:
were you asked to remove this venv too?

Reply:
Yes, it's in the ticket :)

Reply:
@m-dati  yes it is confirmed with @marxin  that we can drop this because we not running openstack tests anymore